core/frontend/src/components/video-manager/VideoControlsDialog.vue

```diff
@@ -160,28 +169,36 @@ export default Vue.extend({
           value = control.configuration.Slider.value
         }
 
-        back_axios({
-          method: 'post',
-          url: `${video.API_URL}/v4l`,
-          timeout: 10000,
-          data: {
-            device: this.device.source,
-            v4l_id: control.id,
-            value,
-          },
-        }).catch((error) => {
+        try {
+          await back_axios({
+            method: 'post',
+            url: `${video.API_URL}/v4l`,
+            timeout: 10000,
+            data: {
+              device: this.device.source,
+              v4l_id: control.id,
+              value,
+            },
+          })
+        } catch (error) {
           const message = `Could not update value on ${control.name} control: ${error}.`
           notifier.pushError('CONTROL_VALUE_UPDATE_FAIL', message)
-        })
-      }
+        } finally {
+          completedControls += 1
```

Comment:
How can `completedControls` be equal to `totalControls` if it only runs once ?

Reply:
Here, `finally` runs for every request, so with the pre-incremented `completedControls`, it will be equal to `totalControls` when all requests are finished.

Reply:
I missed the loop, thanks.